sagemaker-pipelines/step-decorator/computer-vision-examples/computer-vision-pipeline.ipynb

```diff
@@ -0,0 +1,1101 @@
+{
```

Comment:
<p>We can simply delete the pipeline via <code>pipeline.delete()</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/4543/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='48'/>

Reply:
Good point, will update it.